File: csharp/MovieRental/Movie.cs
using System;

namespace MovieRental
{
    public class Movie
    {
        public const int CHILDRENS = 2;
        public const int NEW_RELEASE = 1;
        public const int REGULAR = 0;

        private String _title;
        private int _priceCode;

        public Movie(String title, int priceCode)
        {
            _title = title;
            _priceCode = priceCode;
        }

        public int GetPriceCode()
        {
            return _priceCode;
        }

        public void setPriceCode(int arg)
        {
            _priceCode = arg;
        }
        public String GetTitle()
        {
            return _title;
        }
    }
}

File: scala/src/main/scala/movierental/Movie.scala
package movierental

class Movie(var title: String, var priceCode: Int) {

  def getPriceCode(): Int = {
    return priceCode
  }

  def setPriceCode(arg: Int): Unit = {
    this.priceCode = arg
  }

  def getTitle(): String = {
    return title
  }


}

object Movie {
  val CHILDRENS: Int = 2
  val NEW_RELEASE: Int = 1
  val REGULAR: Int = 0
}
